game/renderware_binary_stream.ksy

```diff
@@ -162,16 +166,23 @@ types:
         repeat: expr
         repeat-expr: _parent.num_vertices
         if: _parent.is_prelit
-      - id: tex_coords
-        type: tex_coord
+      - id: uv_layers
+        type: uv_layer(_parent.num_vertices)
         repeat: expr
-        repeat-expr: _parent.num_vertices
-        if: _parent.is_textured or _parent.is_textured2
-        # FIXME: repeated for number of texture sets
+        repeat-expr: _parent.uv_layer_count
```

Comment:
Should be `num_uv_layers` (see https://doc.kaitai.io/ksy_style_guide.html#attr-id).

```sh
sed -i -e 's/\<uv_layer_count\>/num_uv_layers/g' -e 's/\<uv_layer_count_raw\>/num_uv_layers_raw/g' renderware_binary_stream.ksy
```